src/main/java/com/neuvector/model/NVScanner.java

```diff
@@ -10,6 +10,8 @@ public class NVScanner {
     String nvMountPath;
     Logger log;
     Boolean bindMountShared = false;
+    String runtime;
+    String socketMapping;
```

Comment:
I think it's safer to set this two new fields to their default values.

Reply:
Done in 232194fe45841f1e6ee155f0d3a86254cc1c8291 and c6511d28c303e030b3c334f7adc6621d972ca7e7

Reply:
What I meant is this:
```suggestion
    String socketMapping = Scanner.SOCKET_MAPPING;
```
Same for `runtime`. If any constructor that doesn't set them is called, they will be `null`.

Reply:
Sorry, here: 999d9bace00dd0786b0562b7e01e3922b63ff834